bigquery/api/load_data_by_post.py

```diff
@@ -78,12 +87,14 @@ def load_data(schema_path, data_path, project_id, dataset_id, table_id):
         projectId=job['jobReference']['projectId'],
         jobId=job['jobReference']['jobId'])
 
+    # Poll the job until it finishes.
     while True:
         result = status_request.execute(num_retries=2)
 
         if result['status']['state'] == 'DONE':
-            if 'errorResult' in result['status']:
-                raise RuntimeError(result['status']['errorResult'])
```

Comment:
Yeah - when I was repro'ing, the errorResult just gave the last error, which was pretty uninformative ("too many errors encountered" or some such). It was more helpful to print out all the errors.